yarn-project/simulator/src/public/avm_executor.test.ts

```diff
@@ -43,7 +43,7 @@ describe('AVM WitGen and Proof Generation', () => {
     //    new Add(/*indirect=*/ 0, TypeTag.FIELD, /*aOffset=*/ 0, /*bOffset=*/ 1, /*dstOffset=*/ 2),
     //    new Return(/*indirect=*/ 0, /*returnOffset=*/ 2, /*copySize=*/ 1),
     // ]);
-    const bytecode: Buffer = Buffer.from('IAAAAAAAAAAAAgAAAAAAAAYAAAAAAAAAAQAAAAI4AAAAAAIAAAAB', 'base64');
+    const bytecode: Buffer = Buffer.from('IAAAAAAAAAAAAgAAAAAAAAYAAAAAAAAAAQAAAAI5AAAAAAIAAAAB', 'base64');
```

Comment:
why cant we use encode to bytecode here again?

Reply:
it was to avoid having the dependency to the instructions and bytecode encoding. In any case Ilyas will remove this test as soon as the one below works. (maybe tomorrow?)